.vib/mariadb/ginkgo/mariadb_suite_test.go

```diff
@@ -40,6 +40,21 @@ func TestMariaDB(t *testing.T) {
 }
 
 func createJob(ctx context.Context, c kubernetes.Interface, name, port, image, stmt string) error {
+	securityContext := &v1.SecurityContext{
+		Privileged:               new(bool),
+		AllowPrivilegeEscalation: new(bool),
+		RunAsNonRoot:             new(bool),
+		Capabilities: &v1.Capabilities{
+			Drop: []v1.Capability{"ALL"},
+		},
+		SeccompProfile: &v1.SeccompProfile{
+			Type: "RuntimeDefault",
+		},
+	}
+
+	*securityContext.Privileged = false
+	*securityContext.AllowPrivilegeEscalation = false
+	*securityContext.RunAsNonRoot = true
```

Comment:
It wouldn't allow me to put directly the values in the struct declaration above.

Reply:
I have a trick for you that's going to blow your mind 🤯 
```golang
	securityContext := &v1.SecurityContext{
		Privileged:               &[]bool{false}[0],
		AllowPrivilegeEscalation: &[]bool{false}[0],
		RunAsNonRoot:             &[]bool{true}[0],
		Capabilities: &v1.Capabilities{
			Drop: []v1.Capability{"ALL"},
		},
		SeccompProfile: &v1.SeccompProfile{
			Type: "RuntimeDefault",
		},
	}
```